htdocs/index.html

```diff
@@ -73,9 +73,12 @@ <h2>How do I get rid of systemd?</h2>
 			<li><a href="https://artixlinux.org/"><img src="img/artix.png" alt="Artix" /></a></li>
 			<li><a href="https://www.gnu.org/software/guix/"><img src="img/guix.png" alt="Guix" /></a></li>
 			<li><a href="https://gentoo.org/"><img src="img/gentoo.png" alt="Gentoo" /></a></li>
+			<br>
```

Comment:
I suggest that the line break gets removed as it likely will break weirdly on mobile devices.

Reply:
I've forgot that. Done.